### PR TITLE
ISAICP-6538: Rename EUA to OE Bootstrap Theme

### LINKED ISSUE #2: ISAICP-6538: Rename eua_theme to oe_bootstrap_theme
Joinup ticket https://citnet.tech.ec.europa.eu/CITnet/jira/browse/ISAICP-6538

### PR DESCRIPTION
**Pull request by claudiu-cristea**
Fixes #2

Jira issue(s):
- [ISAICP-6538](https://citnet.tech.ec.europa.eu/CITnet/jira/browse/ISAICP-6538)

### CODE REVIEW

**Review by pfrenssen — DISMISSED** [review]

File: oe_bootstrap_theme.libraries.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 style:
   version: VERSION
   js:
-    assets/js/eua.script.js: {}
+    assets/js/oe_boostrap_theme.script.js: {}
```
Comment (vever001) [inline oe_bootstrap_theme.libraries.yml]: Typo, missing t: oe_boostrap_theme > oe_bootstrap_theme

File: oe_bootstrap_theme.libraries.yml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
   css:
     theme:
-      assets/css/eua.style.css: {}
+      assets/css/oe_boostrap_theme.style.css: {}
```
Comment (vever001) [inline oe_bootstrap_theme.libraries.yml]: Typo, missing t: oe_boostrap_theme > oe_bootstrap_theme

File: src/Theme/OpenEuropaBoostrapThemePreRender.php [hunk 3]
```diff
@@ -1,16 +1,16 @@
 <?php
```
Comment (vever001) [inline src/Theme/OpenEuropaBoostrapThemePreRender.php]: Same here for the filename, class and usages:
- OpenEuropaBoostrapThemePreRender.php > OpenEuropaBootstrapThemePreRender.php
- class OpenEuropaBoostrapThemePreRender > class OpenEuropaBootstrapThemePreRender
- and calls to that class

**Review by vever001 — CHANGES_REQUESTED** [review]
I found some typos, for the rest it looks good.

**Comment by claudiu-cristea** [comment]
@vever001 it seems that I was hungry and I ate that "t" :) Fixed the remarks, thank you

**Review by vever001 — APPROVED** [review]
Looks good to me

**Review by idimopoulos — APPROVED** [review]

**Review by brummbar — APPROVED** [review]
Approving on behalf of @vever001 .

**Pull request merged**